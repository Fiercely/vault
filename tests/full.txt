VAULT=SECRETS
MADE=BY
COMPANY=FIERCELY
AUTHOR=ILHICAS
