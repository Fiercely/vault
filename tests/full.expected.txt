U2FsdGVkX1/ThWuDwJyiDZQAlYVQ3X2EM8Rx+f/OEkCY87pBztwKDkuI1qpq2myD
FcsU/ikyb+YYpgLv4nF7OE0gPFByJraVZz5E8tBtXOg=
